test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala

```diff
@@ -230,7 +230,7 @@ class SubsidyControllerSpec
           findHintText shouldBe s"For example, 15 3 ${year}"
         }
 
-        "legend is displayed as H1 in fieldset component" in {
+        "H1 and paragraph both have legend both displayed in fieldset component" in {
```

Comment:
The h1 does not need to be in the fieldset

Reply:
can you expand on this one?